README.md

```diff
@@ -462,6 +464,16 @@ and every `send_mail` call will appear instantly in the UI.
 > EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 > ```
 
+### Installing The Required LaTeX Packages For Local Development
+
+In order to generate the equations in Wagtail using LaTeX strings via Matplotlib (Non-JS Equations), we will need to use `MacPorts` to install the following packages `texlive-latex-extra`.
+
+As a prerequisite you will have to have [MacPorts](https://www.macports.org/install.php) installed.
+
+```bash
+sudo port install texlive-latex-extra
```

Comment:
For me, it still doesn't work, as it complains about fonts when trying to generate an equation.
I get `! LaTeX Error: File 'type1ec.sty' not found.`

Installing the following fixed it: `sudo port install texlive-fonts-recommended`

Reply:
Still no luck, the pkg installer already supposedly runs `sudo port -v selfupdate`, but I have tried it manually, then reinstalled `texlive-latex-extra` and still no joy. Can you try the following formula and see if it works `$$\sum_{i=0}^n i^2 = \frac{(n^2+n)(2n+1)}{6}$$`.

Reply:
I'll do a fresh install from scratch and also got @AdamHawtin to test the instructions to see why we are experiencing different behaviours. Let's see comes out of it

Reply:
Done! [6cd9231](https://github.com/ONSdigital/dis-wagtail/pull/306/commits/6cd9231266845164af17ba3e7b237e6fc6a924a2)